### DISCUSSION TITLE
Tools for annotating/categorising pathways

### DISCUSSION THREAD

**Discussion by petermr** [wikipathways/GPML-Repo#33] [General]
# Background. 

We are working with Alex and others to build tools to extract terpene metabolism pathways from the images literature. Alex has created a set of images 
https://gladstone-bioinformatics.shinyapps.io/shiny-terpeneplant/
and pointed to a curation tool.
https://github.com/wikipathways/pfocr-curation

We now have new interns (including Ananya and Vaishnavi and maybe others) who are keen to start manual annotations for ground truth. They'll need to create their own classification categories and they'd like to know what your tool does and I'll point them at this list.

**Comment by AlexanderPico** [comment]
Great! I added some docs to the github repo (https://github.com/wikipathways/pfocr-curation). Please start by reading those and attempting to run the tool. If you run into any problems, let me know here, there or by email.

Feel free to play with the tool and use it in "Training" mode without concern. This is the default setting for the tool. When ready to do real classification, let's coordinate on any new categories (and GUI buttons) you need and on splitting up the work to avoid overlap.
